website/src/components/Tasks/CreateTask.tsx

```diff
@@ -55,6 +57,7 @@ export const CreateTask = ({
                 {t(getTypeSafei18nKey(`tasks:${taskType.id}.instruction`))}
               </Text>
             )}
+            <Text color={tipColor}>Tip: Use <Kbd>ctrl</Kbd>+<Kbd>Enter</Kbd> to Review and Submit</Text>
```

Comment:
please extract to localization, also we need to check how it works on macs, we also need to hide this text on mobile.

Reply:
Thanks for the feedback. Will swing back around on this tomorrow (or anyone else interested can jump in).

I do also have a Mac to test that `e.metaKey` works as it's supposed to.

Reply:
@AbdBarho how are we doing translations for localization? Simply Google Translate or?

Reply:
Working on Mac: 
<img width="767" alt="image" src="https://user-images.githubusercontent.com/12889451/217125054-2df4a8b7-ebc3-480f-90ef-fbffa2d0b80d.png">

Reply:
Great! no need to worry about translating the text now, my comment with regarding not hard-coding the text in the react component, but just moving it the locale json file, so that it can be translated easily later.